src/test/test-ipcrm.c

```diff
@@ -28,9 +28,11 @@ int main(int argc, char *argv[]) {
 
         r = get_user_creds(&name, &uid, NULL, NULL, NULL);
         if (r < 0) {
-                log_error_errno(r, "Failed to resolve \"%s\": %m", name);
-                return EXIT_FAILURE;
+                log_full_errno(r == -ESRCH ? LOG_NOTICE : LOG_ERR,
+                               r, "Failed to resolve \"%s\": %m", name);
+                return r == -ESRCH ? EXIT_TEST_SKIP : EXIT_FAILURE;
         }
 
-        return clean_ipc_by_uid(uid) < 0 ? EXIT_FAILURE : EXIT_SUCCESS;
+        r = clean_ipc_by_uid(uid);
+        return  r < 0 ? EXIT_FAILURE : EXIT_SUCCESS;
```

Comment:
Entirely bikeshedding, but out of interest: Is that preferred code style these days?